src/shipping/src/telemetry_conf.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I had a comment above saying - if you can't init OTel and you continue, what's your fallback for logs? I reckon you could make sure to add a console appender to the log crate or something in that case, you'll just have to structure the error handling a bit.